File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String ans = "";
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        if (a + b + c != 180) {
            ans = "Error";
        } else {
            if (a == b && b == c) {
                ans = "Equilateral";
            } else if (a == b || b == c || c == a) {
                ans = "Isosceles";
            } else {
                ans = "Scalene";
            }
        }
        System.out.println(ans);
    }
}

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.cc
#include <stdio.h>

int main() {
  int a, b, c;
  scanf("%d\n%d\n%d", &a, &b, &c);
  if (a == 60 && a == b && a == c) printf("Equilateral");
  else if (a + b + c == 180) {
    if (a == b || b == c || a == c) printf("Isosceles");
    else printf("Scalene");
  }
  else printf("Error");
}